packages/opentelemetry-metrics/src/export/ConsoleMetricExporter.ts

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MetricExporter, ReadableMetric } from './types';
+import { ExportResult } from '@opentelemetry/base';
+
+/**
+ * This is implementation of {@link MetricExporter} that prints metrics data to
+ * the console. This class can be used for diagnostic purposes.
+ */
+export class ConsoleMetricExporter implements MetricExporter {
+  export(
+    metrics: ReadableMetric[],
+    resultCallback: (result: ExportResult) => void
+  ): void {
+    for (const metric of metrics) {
+      const descriptor = metric.descriptor;
+      const timeseries = metric.timeseries;
+      console.log({
+        name: descriptor.name,
+        description: descriptor.description,
+      });
+      for (const ts of timeseries) {
+        for (const point of ts.points) {
+          console.log({ labels: ts.labelValues, value: point.value });
```

Comment:
sidequestion: Is showing the label keys somewhere something that should be handled by this (or any) exporter? In case there are label values which could be interpreted differently based on which key it belongs to

Reply:
Can we zip labelKeys and labelValues together into a `{key: value}` object for the exporter?

```typescript
keys.map((k, i) => [k, values[i]]).reduce((p, c) => ({ ...p, [c[0]]: c[1] }), {});
```

Reply:
Done, ptal 9b5c3261a1f70b34735daabba88e43cf57ad3633